<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VerProceso</class>
 <widget class="QWidget" name="VerProceso">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>328</width>
    <height>515</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Ver Proceso</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabProceso">
      <attribute name="title">
       <string>Info. Proceso</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::ExpandingFieldsGrow</enum>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="lbl1">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Demandante:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="lblDemandante">
           <property name="toolTip">
            <string>El nombre del demandante</string>
           </property>
           <property name="statusTip">
            <string>El nombre del demandante</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lbl2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Demandado:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="lblDemandado">
           <property name="toolTip">
            <string>El nombre del demandado</string>
           </property>
           <property name="statusTip">
            <string>El nombre del demandado</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lbl3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Fecha:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lbl4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Juzgado:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="lblJuzgado">
           <property name="toolTip">
            <string>El nombre del juzgado del proceso</string>
           </property>
           <property name="statusTip">
            <string>El nombre del juzgado del proceso</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="lbl5">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Radicado:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="lblRadicado">
           <property name="toolTip">
            <string>El radicado del proceso</string>
           </property>
           <property name="statusTip">
            <string>El radicado del proceso</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="lblRadicadoUnico">
           <property name="toolTip">
            <string>El radicado único del proceso</string>
           </property>
           <property name="statusTip">
            <string>El radicado único del proceso</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="lbl7">
           <property name="text">
            <string>Categoría:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="lbl6">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Radicado Único:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="lbl8">
           <property name="text">
            <string>Estado:</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="lbl9">
           <property name="text">
            <string>Tipo:</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="lbl10">
           <property name="text">
            <string>Notas:</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="lbl11">
           <property name="text">
            <string>Prioridad:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="lblCategoria">
           <property name="toolTip">
            <string>La categoría donde está el proceso</string>
           </property>
           <property name="statusTip">
            <string>La categoría donde está el proceso</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLabel" name="lblEstado">
           <property name="toolTip">
            <string>El estado del proceso</string>
           </property>
           <property name="statusTip">
            <string>El estado del proceso</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLabel" name="lblTipo">
           <property name="toolTip">
            <string>El tipo de proceso</string>
           </property>
           <property name="statusTip">
            <string>El tipo de proceso</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QLabel" name="lblNotas">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Las notas sobre el proceso</string>
           </property>
           <property name="statusTip">
            <string>Las notas sobre el proceso</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QLabel" name="lblPrioridad">
           <property name="toolTip">
            <string>La prioridad del proceso</string>
           </property>
           <property name="statusTip">
            <string>La prioridad del proceso</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QDateTimeEdit" name="dteFecha">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>La fecha de creación del proceso</string>
           </property>
           <property name="statusTip">
            <string>La fecha de creación del proceso</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
